### DISCUSSION TITLE
Support for MKS DLC32 2.1

### DISCUSSION THREAD

**Discussion by padcom** [Paciente8159/uCNC#697] [Q&A] [answered]
Is there a way to compile µCNC for the Makerbase DLC32 2.1 board?

https://github.com/makerbase-mks/MKS-DLC32/blob/main/MKS-DLC32-main/hardware/MKS%20DLC32%20V2.1_003/MKS%20DLC32%20V2.1_003%20PIN.pdf

**Comment by Paciente8159** [comment, marked as answer]
The project already as a preconfigured setup for that board.
The easiest way is to use VSCode and PlatformIO. Select the environment for that board and compile and upload.

If you need help configuring the environment let me know.
